modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+import {navigateTo, navigateToACSitesPageViaURL} from './utils/navigation';
+import {CardSelectors} from './utils/selectors';
+import {changeTimeFilter} from './utils/time-filter';
+import {liferayConfig} from '../../liferay.config';
+import {createChannel} from './utils/channel';
+import {createIndividuals} from './utils/individuals';
+import {viewNameOnTableList} from './utils/utils';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+test(
+	'Verify that clicking the "All Search Terms" link redirects the user to the Search Terms tab with the same time filter previously set.',
+
+	{
+		tag: '@LPD-31557',
```

Comment:
Hey @fmarcio, We can put the ID of the Story or BUG that originated this automation.
```suggestion
		tag: '@LPD-26181',
```

Reply:
done